pkg/controllers/status/controller.go

```diff
@@ -488,9 +498,8 @@ func (s *KubeFedStatusController) clusterStatuses(
 			s.typeConfig.GetTargetType(),
 		)
 		if err != nil {
-			wrappedErr := errors.Wrapf(err, "Failed to get %s %q from cluster %q", targetKind, key, clusterName)
-			runtime.HandleError(wrappedErr)
-			return nil, wrappedErr
+			keyedLogger.WithValues("target-kind", targetKind, "cluster-name", clusterName).Error(err, "Failed to get object from cluster")
+			return nil, errors.Wrapf(err, "Failed to get %s %q from cluster %q", targetKind, key, clusterName)
```

Comment:
Prefer `fmt.Errorf` over `errors.Wrapf`

Reply:
done, pls take review.

Reply:
Let's address 'errors' once and for all next time.